CameraOverlay/.env

```diff
@@ -0,0 +1 @@
+MHP_CAMERA=primary
```

Comment:
On second thought, let's not commit this file. Please document in the `README` on how to create this file and what to put in it. (See MHP DAS Web Server repo)

Reply:
sure i'll do that :)